localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -14,41 +14,47 @@
  ##############
  # Tribal Era #
  ##############
-
+ 
  innovation_amphibious:0 "Amphibious"
  innovation_amphibious_desc:0 "We surface on land only when we please, but being natives to the water we can travel the seas at ease."
 
- innovation_sea_faring:0 "Sea Faring"
- innovation_sea_faring_desc:0 "Living on the coasts gave us a natural advantage in not only taming, but conquering the seas and those who dare to challenge us."
-
- innovation_lorewalking:0 "Lorewalking"
- innovation_lorewalking_desc:0 "Our ability to proficiently record history help us find ourselves through the lore of our ancestors."
-
```

Comment:
So yeah, as it was said in the dev, don't move loc lines. It may cause chaos in Russian and French localization and add additional work for translators. So bring back old lines to old place and move new lines to the end.

Reply:
should I do this for german as well since it has the exact layout as the english loc?

Reply:
> should I do this for german as well since it has the exact layout as the english loc?

Yes.